__tests__/angular-json/regular-angular10_1-and-up-ngpackagr-builder-angular-json.ts

```diff
@@ -0,0 +1,4 @@
+import {AngularJson} from '../../src/types/angular-json';
+import {regularAngularJson} from './regular-angular-json';
+
+export const regularAngular10_1AndUpNgPackagrBuilderAngularJson: AngularJson = regularAngularJson;
```

Comment:
question: Why do we have two files when this export is the same as `regular-angular-json.ts`?

Reply:
Yes, I probably should change this a bit.